src/cli/rustup_mode.rs

```diff
@@ -245,8 +242,11 @@ pub fn cli() -> App<'static, 'static> {
                         .help(TOOLCHAIN_ARG_HELP)
                         .required(false),
                 ),
-        )
-        .subcommand(
+        );
+
+    // We break the app creation here so that rustfmt can cope
+    // If rustfmt ceases to format this block, break it up further
```

Comment:
Is there an issue about that in rustfmt?

Reply:
This is indeed a known issue in rustfmt though I was told it's unlikely to be resolved before 2.0